recipes/rdflib-jsonld/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "rdflib-jsonld" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 144774786a2fc7de09b24a9309cbcccc78bc48b152536d2ea1c1df2ad715bc2d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Comment:
can this be noarch?

Reply:
the setup.py defines custom entry points here: https://github.com/RDFLib/rdflib-jsonld/blob/master/setup.py#L105

according to regro-bot, I would have to put them in meta.yaml, but I am not sure if that's at all possible.
> If entrypoints are in setup.py, they are listed in meta.yaml

Reply:
Could you list the entry points like in this PR? https://github.com/conda-forge/staged-recipes/pull/7259/files#diff-15b86f6473019521b86e8a08175d7181R16

Reply:
In the case you linked, a console script named `iapsg` is added which calls apsg.shell.main, right? But the entrypoints inrdflib-jsonld are not console scripts. That's why I am not sure whether/how this can be done.

Reply:
That's fine.